3scaleAdapter/pkg/threescale/threescale_test.go

```diff
@@ -62,27 +67,43 @@ func TestHandleAuthorization(t *testing.T) {
 			},
 		},
 		{
+			name: "Test fail - Non 2xx status code from 3scale backend",
 			params: pb.Params{
 				ServiceId:   "123",
 				SystemUrl:   "https://www.fake-system.3scale.net",
 				AccessToken: "expect7",
 			},
 			expectStatus: 7,
 			action: &authorization.ActionMsg{
-				Path: "/test?user_key=invalid-backend-resp",
+				Path:   "/test?user_key=invalid-backend-resp",
+				Method: "get",
 			},
 		},
 		{
+			name: "Test fail - non-matching mapping rule",
 			params: pb.Params{
 				ServiceId:   "123",
 				SystemUrl:   "https://www.fake-system.3scale.net",
 				AccessToken: "happy-path",
 			},
-			expectStatus: 0,
+			expectStatus: 7,
```

Comment:
@unleashed I've modified this test now to match the case where we have not matched on method and path.

Reply:
what is the method used here?

Reply:
None, so the methods is empty. The mock returns a get and if it doesn't match that then it will fail. I'll add a `POST` here for clarity

Reply:
Done